### PR TITLE
App Logs Documentation

### PR DESCRIPTION
**Pull request by aoleszyn**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+1. On Sauce Labs, in the left navigation, click **App Mangement**.
+
+<br/><img src={useBaseUrl('img/mobile-apps/app-logs5.png')} alt="App Logs" width="300"/>
+
+2. Upload your mobile app to Sauce Labs.
+
+  <br/><img src={useBaseUrl('img/mobile-apps/app-logs-8.png')} alt="App Logs" width="700"/> 
+
+3. After you’ve uploaded your app, hover your mouse over your app, then select **Settings**.
```
Comment (aliciakrebs) [inline docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md:70]:
```suggestion
3. After you’ve uploaded your app, hover over your app, then select **Settings**.
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+id: app-logs
+title: App Logs
+sidebar_label: App Logs
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+App Logs is functionality that provides you with a detailed view of logs for mobile devices and desktop browsers. This feature provides you with visibility into system-level events, errors, and warnings that occur during testing, making it easier to troubleshoot and diagnose issues.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md:9]: only mobile devices

Comment (aoleszyn) [inline docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md:9]:
```suggestion
App Logs is functionality that provides you with a detailed view of logs for your mobile apps. This feature provides you with visibility into events, errors, and warnings that occur during testing, making it easier to troubleshoot and diagnose issues.
```

File: docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   <br/><img src={useBaseUrl('img/mobile-apps/app-logs-1.png')} alt="App Logs" width="300"/>
+
+2. Click on the device name for the test session that needs debugging.
+<br/><img src={useBaseUrl('img/mobile-apps/app-logs-2.png')} alt="App Logs" width="800"/>
+
+3. Click the **Logs** tab and select **device.log** from the drop down menu.
+
+<br/><img src={useBaseUrl('img/mobile-apps/app-logs-3.png')} alt="App Logs" width="800"/>
+
+The App Logs feature includes different log types, such as errors, warnings, infos, and verbose Logs. Each log type provides a different view of the device logs, allowing you to filter and focus on specific types of information.
```
Comment (aoleszyn) [inline docs/mobile-apps/features/mobile-app-diagnostics/app-logs.md:36]:
```suggestion
The App Logs feature includes different log types, such as errors, warnings, infos, and verbose Logs. Each log type provides a different view of the application logs, allowing you to filter and focus on specific types of information.
```

**Pull request merged**